src/components/common/YourCompany/OfficeAddresses.vue

```diff
@@ -575,7 +575,7 @@ import { Component, Mixins, Prop, Watch } from 'vue-property-decorator'
 import { Action, Getter } from 'pinia-class'
 import { isEmpty, isEqual } from 'lodash'
 import { DefaultAddressSchema, InBcCanadaAddressSchema } from '@/schemas/'
-import BaseAddress from 'sbc-common-components/src/components/BaseAddress.vue'
+import { BaseAddress } from '@bcrs-shared-components/base-address'
```

Comment:
Please test and/or list in ticket all the BaseAddress uses, and also the PDF uploads.

Reply:
Tested. Everything is working smoothly. I'll also add a note for Riyaz just in case.